src/components/CustomNodeWidget.tsx

```diff
@@ -295,7 +332,7 @@ export class CustomNodeWidget extends React.Component<DefaultNodeProps> {
                                 />
                             </label>
                         </S.Title>
-                        <S.Ports>
+                        <S.Ports>{/*aici se pun intrarile*/}
```

Comment:
I think this comment should be removed :-)

Reply:
Solved